R/checks.R

```diff
@@ -158,22 +154,21 @@ check_parameters <- function(wflow, pset = NULL, data, grid_names = character(0)
 
   if (needs_finalization(pset, grid_names)) {
     if (tune_recipe) {
-      rlang::abort(
-        paste(
-          "Some model parameters require finalization but there are recipe",
-          "parameters that require tuning. Please use ",
-          "`extract_parameter_set_dials()` to set parameter ranges ",
-          "manually and supply the output to the `param_info` argument."
+      cli::cli_abort(
+        c(
+          "Some model parameters require finalization but there are recipe
+           parameters that require tuning.",
+          "i" = "Please use {.fn extract_parameter_set_dials} to set parameter
+           ranges manually and supply the output to the {.arg param_info}
+           argument."
         )
       )
     }
-    msg <- "Creating pre-processing data to finalize unknown parameter"
     unk_names <- pset$id[unk]
-    if (length(unk_names) == 1) {
-      msg <- paste0(msg, ": ", unk_names)
-    } else {
-      msg <- paste0(msg, "s: ", paste0("'", unk_names, "'", collapse = ", "))
-    }
+    num_unk <- length(unk_names)
+    msg <- cli::format_inline(
+      "Creating pre-processing data to finalize {num_unk} unknown parameter{?s}: {.val {unk_names}}"
+    )
```

Comment:
```suggestion
    msg <- 
      "Creating pre-processing data to finalize unknown parameter{?s} {.val {unk_names}}."
```

Since this is a very common message I am going to be fussy about my enum-after-parens pet peeve--feel free to ignore that part of this suggestion if you please🤪